client/src/js/base/Tooltip.js

```diff
@@ -1,35 +1,16 @@
 import React from "react";
-import PropTypes from "prop-types";
+import styled from "styled-components";
+import { Tooltip } from "react-tippy";
```

Comment:
You can rename imports the following way (prevents clash with base component name `Tooltip`):
```js
import { Tooltip as TippyTooltip } from "react-tippy";
```